### PR TITLE
Layouts, skeletons, presistent theme

### PR DESCRIPTION
**Pull request by jvonasek**

### CODE REVIEW

File: apps/main/src/modules/layout/components/Header.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import { Flex, Toggle } from "@galacticcouncil/ui/components"
+import { useTheme } from "@galacticcouncil/ui/theme"
+import { getToken } from "@galacticcouncil/ui/utils"
+import { Link } from "@tanstack/react-router"
+
+const NAVIGTION = [
```
Comment (vkulinich-cl) [inline apps/main/src/modules/layout/components/Header.tsx:6]: I'd move it out from Header.tsx component. Let's have a utils file for things like those

File: packages/ui/src/components/Paper/Paper.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { Box, BoxProps } from "@/components/Box"
+import { getToken } from "@/utils"
+
+export type PaperProps = BoxProps
+
+export const Paper: React.FC<PaperProps> = (props) => {
```
Comment (vkulinich-cl) [inline packages/ui/src/components/Paper/Paper.tsx:6]: Let's have a styled component. It doesn't have any logic

File: packages/ui/src/components/Skeleton/Skeleton.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import SkeletonPrimitive from "react-loading-skeleton"
+
+import { useTheme } from "@/theme"
+
+export type SkeletonProps = React.ComponentPropsWithoutRef<
+  typeof SkeletonPrimitive
+>
+
+export const Skeleton: React.FC<SkeletonProps> = (props) => {
+  const { themeProps } = useTheme()
```
Comment (vkulinich-cl) [inline packages/ui/src/components/Skeleton/Skeleton.tsx:12]: what's a difference between getToken from utils file and themeProps from the hook?

File: apps/main/src/modules/layout/components/Header.tsx (hunk 1 above)
Comment (jvonasek) [reply apps/main/src/modules/layout/components/Header.tsx:6]: its just a mockup for now, can be done when implementing real navigation

File: packages/ui/src/components/Skeleton/Skeleton.tsx (hunk 3 above)
Comment (jvonasek) [reply packages/ui/src/components/Skeleton/Skeleton.tsx:12]: getToken can be used only in SX prop or other styled-components props like our Box component.

Skeleton component is from installed package react-loading-skeleton and the props baseColor and highlightColor dont support that

**Review by vkulinich-cl — APPROVED** [review]

**Pull request merged**